api/src/main/resources/workbench.yaml

```diff
@@ -888,6 +888,228 @@ paths:
           schema:
             $ref: "#/definitions/CohortSummaryListResponse"
 
+
+  # Data Browser #######################################################################
+  /api/v1/databrowser/search-concepts:
+    get:
+      security: []
+      tags:
+        - dataBrowser
+
+      description: Searches for concepts in concept table with search string and filters
+      operationId: "getConceptsSearch"
+      parameters:
+        - in: query
+          name: concept_name
+          type: string
+          required: false
+          description: concept name search string
+        - in: query
+          name: standard_concept
+          type: string
+          required: false
+          description: standard concept filter
+        - in: query
+          name: concept_code
+          type: string
+          required: false
+          description: concept code  filter
+        - in: query
+          name: vocabulary_id
+          type: string
+          required: false
+          description: vocabulary id filter
+        - in: query
+          name: domain_id
+          type: string
+          required: false
+          description: vocabulary id filter
+      responses:
+        "200":
+          description: A collection of concepts
+          schema:
+            $ref: "#/definitions/ConceptListResponse"
+
+  /api/v1/databrowser/analysis-results:
+    get:
+      security: []
+      tags:
+        - dataBrowser
+
+      description: Gets results for an analysis id and stratum
+      operationId: "getAnalysisResults"
+      parameters:
+        - in: query
+          name: analysis_id
+          type: integer
+          format: int64
+          required: true
+          description: analysis_id
+        - in: query
```

Comment:
See other comment about exposing Achilles schema in API.

Reply:
fixed